source/features/latest-tag-button.tsx

```diff
@@ -13,6 +14,21 @@ import {groupButtons} from '../github-helpers/group-buttons';
 import getDefaultBranch from '../github-helpers/get-default-branch';
 import {buildRepoURL, getCurrentCommittish, getLatestVersionTag, getRepo} from '../github-helpers';
 
+// eslint-disable-next-line import/prefer-default-export
+export async function addAfterBranchSelector(button: Element): Promise<void> {
+	const branchSelector = (await elementReady('#branch-select-menu', {waitForChildren: false}))!;
+	const wrapper = branchSelector.closest('.position-relative')!;
+	button.classList.add('mt-md-2', 'mt-lg-0');
+	wrapper.append(button);
+	if (wrapper.classList.contains('rgh-button-added')) {
+		return;
+	}
```

Comment:
Seems unnecessary, they all add classes afterwards, it's not a huge problem to run them again